python /usr/local/bin/
r /usr/lib/R


/usr/lib/R/bin/R: line 250: /usr/lib/R/etc/ldpaths: No such file or directory
/usr/lib/R/bin/exec/R: error while loading shared libraries: libR.so: cannot open shared object file: No such file or directory

python: error while loading shared libraries: libpython3.9.so.1.0: cannot open shared object file: No such file or directory

---
/usr/lib/R/bin/exec/R: error while loading shared libraries: libblas.so.3: cannot open shared object file: No such file or directory

python: error while loading shared libraries: libpython3.9.so.1.0: cannot open shared object file: No such file or directory



-----
root@949306bf5b6f:/# ldd /usr/local/bin/python3
        linux-vdso.so.1 (0x00007fff62e9c000)
        libpython3.9.so.1.0 => /usr/local/lib/libpython3.9.so.1.0 (0x00007f7fedfb3000)
        libcrypt.so.1 => /lib/x86_64-linux-gnu/libcrypt.so.1 (0x00007f7fedf79000)
        libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f7fedf58000)
        libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f7fedf53000)
        libutil.so.1 => /lib/x86_64-linux-gnu/libutil.so.1 (0x00007f7fedf4e000)
        libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f7feddcb000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f7fedc08000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f7fee38b000)

/bin/R
root@7504ec2bfccf:/# ldd /bin/Rscript
        linux-vdso.so.1 (0x00007ffda9b68000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f9d9f2dd000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f9d9f4ca000)